DependencyInjection/DoctrineMigrationsExtension.php

```diff
@@ -25,8 +34,95 @@ public function load(array $configs, ContainerBuilder $container) : void
 
         $config = $this->processConfiguration($configuration, $configs);
 
-        foreach ($config as $key => $value) {
-            $container->setParameter($this->getAlias() . '.' . $key, $value);
+        $configurationDefinition = new Definition(MigrationsConfiguration::class);
+        $container->setDefinition('doctrine.migrations.configuration', $configurationDefinition);
+
+        if (! empty($config['name'])) {
+            $configurationDefinition->addMethodCall('setName', [$config['name']]);
+        }
+        if (! empty($config['migrations_paths'])) {
```

Comment:
If the array is empty the foreach will not loop over it, so I don't think there is a point doing this check.